Packs/Core/Playbooks/playbook-Excessive_User_Account_Lockouts_README.md

```diff
@@ -0,0 +1,67 @@
+This playbook is designed to handle the following alerts:
```

Comment:
Change to:  This playbook addresses the following alerts:

Reply:
Changing. As for the missing title - since the template applies to RN and this is the README of the playbook which is derived from the playbook description, it would make sense not to have this title